0.8/docs/devguide/properties.md

```diff
@@ -135,22 +136,27 @@ for more information.
 ## Property name to attribute name mapping {#property-name-mapping}
 
 For data binding, deserializing properties from attributes, and reflecting
-properties back to attributes, {{site.project_title}} must map attribute names to property
-names and the reverse. However, attribute names are case-insensitive and can
-contain dashes, while property names are case-sensitive and cannot contain
-dashes.
+properties back to attributes, {{site.project_title}} maps attribute names to property
+names and the reverse. 
+
+When mapping attribute names to property names:
 
-When mapping attribute names to property names, the general rule is that
-attribute names are converted to lowercase (since the DOM is case-insensitive
-for attribute names).
+*   Attribute names are converted to lowercase property names. For example,
```

Comment:
Hm. I was trying to make this clearer. Here. the _attribute_ name is firstName, and it gets converted to the property name "firstname". (No camel case).

If you want camel case, you need to use a dash in the attribute name. Let me know if you can think of a clearer way to say this.